docs/querying/sql.md

```diff
@@ -871,13 +871,13 @@ Submit your query as the value of a "query" field in the JSON object within the
 {"query" : "SELECT COUNT(*) FROM data_source WHERE foo = 'bar'"}
 ```
 
-##### Request
+##### Request body
       
 |Property|Description|Default|
 |--------|----|-----------|
 |`query`|SQL query string.| none (required)|
 |`resultFormat`|Format of query results. See [Responses](#responses) for details.|`"object"`|
-|`header`|Whether or not to include a header. See [Responses] for details.|`false`|
+|`header`|Whether or not to include a header. See [Responses](#responses) for details.|`false`|
```

Comment:
Maybe clarify further that this means header row for the result set, not having to do with the HTTP header.